diffkemp/semdiff/result.py

```diff
@@ -81,6 +81,17 @@ def to_output_yaml_with_rel_path(self, prefix):
                      "line": call["line"]}
                     for call in self.calls]
 
+        @staticmethod
+        def get_name_and_kind(call):
+            """Returns tuple (name, kind) for a call."""
+            name = call["name"]
+            kind = "function"
+            if " " in name:
+                name, kind = name.split(" ")
+                # kind looks like this "(kind)", extracting it from brackets
+                kind = kind[1:-1]
+            return (name, kind)
```

Comment:
What about this? Slightly shorter and more readable (at least for me)
```suggestion
            split = call["name"].split()
            if len(split) == 1:
                return (split[0], "function")
            # The expected form is "name (kind)"
            return (split[0], split[1][1:-1])
```
Also note that whitespace is the default separator for `split`.